model-optimizer/telemetry/backend/backend.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
Do we plan to make this library common for all the Python components that are going to have a telemetry feature? Or this is just the next step after POC.

Reply:
For the 2021.3 it will be a part of the MO since it is the only component being instrumented. We will change this with the future releases.